### ISSUE TITLE
File Header rule panic with "index out of range"

### ISSUE THREAD

**Issue by freedemster** [mgechev/revive#245]
**Describe the bug**
I got some files without comments and when 'revive' apply 'file-header' rule.

**To Reproduce**
1. I updated revive `go get -u github.com/mgechev/revive`
2. I run it with the following flags & configuration file:

```shell
revive --config ./revive.toml --formatter friendly --exclude ./vendor/... ./... 
```

```toml
ignoreGeneratedHeader = true
severity = "warning"
confidence = 0.8
errorCode = 1
warningCode = 2

[rule.file-header]
  arguments =["Copyright 1999-2019. All rights reserved."]
```

**Expected behavior**
We should not get panic.

**Logs**
```
panic: runtime error: index out of range [0] with length 0

goroutine 54 [running]:
github.com/mgechev/revive/rule.lintFileHeader.Visit(0xc0002d37c0, 0xc00016ce80, 0xc0004840a0, 0xc0003523c0, 0x13c39e0, 0xc00016ce80, 0x8, 0xc000444230)
	/Users/dmitriysemin/go/src/github.com/mgechev/revive/rule/file-header.go:55 +0x436
go/ast.Walk(0x13c0d60, 0xc0003ae320, 0x13c39e0, 0xc00016ce80)
	/usr/local/Cellar/go/1.13.4/libexec/src/go/ast/walk.go:52 +0x66
github.com/mgechev/revive/rule.(*FileHeaderRule).Apply(0x15d5f48, 0xc0002d37c0, 0xc000076f00, 0x1, 0x1, 0xc000335b60, 0xc0002a5980, 0x130e680)
	/Users/dmitriysemin/go/src/github.com/mgechev/revive/rule/file-header.go:37 +0x161
github.com/mgechev/revive/lint.(*File).lint(0xc0002d37c0, 0xc000437390, 0x1, 0x1, 0x1, 0x3fe999999999999a, 0xc00009c990, 0x7, 0xc000079050, 0x1, ...)
	/Users/dmitriysemin/go/src/github.com/mgechev/revive/lint/file.go:108 +0x3b6
github.com/mgechev/revive/lint.(*Package).lint.func1(0xc000437390, 0x1, 0x1, 0x1, 0x3fe999999999999a, 0xc00009c990, 0x7, 0xc000079050, 0x1, 0x2, ...)
	/Users/dmitriysemin/go/src/github.com/mgechev/revive/lint/package.go:173 +0xa8
created by github.com/mgechev/revive/lint.(*Package).lint
	/Users/dmitriysemin/go/src/github.com/mgechev/revive/lint/package.go:172 +0x179

```

**Desktop (please complete the following information):**
 - OS: macOS Mojave 10.14.6
 - Version of Go: 1.13.4
